internal/store/persistentvolumeclaim.go

```diff
@@ -139,6 +139,31 @@ var (
 					}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_persistentvolumeclaim_resized_status_condition",
+			Help: "Information about the status of resizing of persistent volume claim.",
+			Type: metric.Gauge,
+			GenerateFunc: wrapPersistentVolumeClaimFunc(func(p *v1.PersistentVolumeClaim) *metric.Family {
+				ms := make([]*metric.Metric, len(p.Status.Conditions)*len(conditionStatuses))
+
+				for i, c := range p.Status.Conditions {
+					conditionMetrics := addConditionMetrics(c.Status)
```

Comment:
This doesn’t seem specific to volume resizing, in which case the metric name should be more general than what it is right now.

Reply:
Okay, I'll change conditionMetrics to resizingStatusConditionMetrics. That's what you suggesting right?

Reply:
Not really. The metric we're adding here is called `kube_persistentvolumeclaim_resized_status_condition`, but we are iterating over _all_ the conditions, so it's more than just the resizing conditions, no?

Reply:
Correct, I got your point now, it would take all the conditions of PVC. I'll change the metric name to kube_persistentvolumeclaim_status_condition.

Reply:
Or maybe kube_persistentvolumeclaim_condition_status would make more sense

Reply:
Okay I'll use that.

Reply:
made the change